kie-drools-wb-parent/kie-drools-wb-webapp/src/main/webapp/rest-api.jsp

```diff
@@ -41,19 +41,18 @@ Entry point for REST interface is at <%= request.getScheme()+"://"+request.getSe
   <tr><td>/organizationalunits/{organizationalUnitName}</td><td>GET</td><td>get the details for an organizational unit</td></tr>
   <tr><td>/organizationalunits/{organizationalUnitName}</td><td>DELETE</td><td>delete an organizational unit</td></tr>
   <tr><td>/organizationalunits/{organizationalUnitName}/</td><td>POST</td><td>update an existing organizational unit</td></tr>
```

Comment:
I know that this is not a part of this PR, but for this endpoint we should use REST method PUT not POST.

Reply:
As much as I think you're correct, the appropriate action at this point is to open a JIRA issue (possibly for revisiting this whole REST API as "organizationalUnit" is not a term we want to expose to users any longer) and move on.

Collectively these PRs are quite large and they block lots of other important work. We want to avoid increasing the scope of changes to fixing past mistakes in order to merge these in a timely manner while attending to the other bugs that have been reported.

Reply:
Created new issue for invalid REST method https://issues.jboss.org/browse/AF-976